airbyte-integrations/connectors/source-stripe/setup.py

```diff
@@ -5,7 +5,7 @@
 
 from setuptools import find_packages, setup
 
-MAIN_REQUIREMENTS = ["airbyte-cdk==0.52.8", "stripe==2.56.0", "pendulum==2.1.2"]
+MAIN_REQUIREMENTS = ["airbyte-cdk>=0.52.8", "stripe==2.56.0", "pendulum==2.1.2"]
```

Comment:
let's pin the version before releasing to avoid the connector accidentally breaking if we make a breaking change